src/test/java/tests/ConformanceTest.java

```diff
@@ -129,9 +130,16 @@ protected boolean mustBeExpected() {
       if (IRRELEVANT_ANNOTATION_KEYS.contains(detailMessage.messageKey)) {
         return irrelevantAnnotation("Nullable"); // TODO(dpb): Support other annotations.
       }
-      if (detailMessage.messageKey.equals("sourceType")) {
-        return expressionType(
-            fixType(detailMessage.messageArguments.get(0)), detailMessage.messageArguments.get(1));
+      switch (detailMessage.messageKey) {
+        case "sourceType":
+          return expressionType(
+              fixType(detailMessage.messageArguments.get(0)),
+              detailMessage.messageArguments.get(1));
+
+        case "sinkType":
+          return sinkType(
+              fixType(detailMessage.messageArguments.get(0)),
+              detailMessage.messageArguments.get(1).replaceFirst("^[^.]+\\.", ""));
```

Comment:
Is this stripping a class and package name? It's probably worth a comment.

Also, would it make sense to do that work inside the checker, rather than when the test consumes the results?

Reply:
It's stripping just the simple name of the class, which is what CF emits by default.

My feeling is that there's a judgment call about where to adapt the checker to the conformance tests. I'm leaning on basic enabling of sufficient diagnostics in the checker, and string massaging in the adapter.

Reply:
Mostly it just feel strange to me that we'd output data that forces ourselves to munge it. And maybe hypothetically it would be nice to be able to run the checker directly in this debugging mode and see the same simple output that the conformance tests produce.

But what you have is fine.